.gitignore

```diff
@@ -44,6 +44,7 @@ doxygen/
 .project
 .settings/
 .idea
+.vscode
```

Comment:
Please remove this line, there are already files checked into this directory.

Reply:
removed